pyproject.toml

```diff
@@ -0,0 +1,77 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+
+[project]
+name = "pastas"
+version = "0.22.0b"
+description = "Python package to perform time series analysis of hydrological time series."
+license = { file = "LICENSE" }
+readme = "README.rst"
+authors = [{ name = "Collenteur et al. 2019" }]
+maintainers = [
+    { name = "R.A. Collenteur", email = "raoulcollenteur@gmail.com" },
+    { name = "M. Bakker", email = "markbak@gmail.com" },
+    { name = "R. Calje", email = "r.calje@artesia-water.nl" },
+    { name = "F. Schaars", email = "f.schaars@artesia-water.nl" },
+    { name = "D.A. Brakenhoff", email = "d.brakenhoff@artesia-water.nl" },
+    { name = "O. Ebbens", email = "o.ebbens@artesia-water.nl" },
+    { name = "M.A. Vonk", email = "m.vonk@artesia-water.nl" },
+]
+
+requires-python = ">= 3.7"
+dependencies = [
+    "numpy >= 1.17, < 1.24",
+    "matplotlib >= 3.1",
+    "pandas >= 1.1",
+    "scipy >= 1.3",
+]
+
+keywords = ["hydrology", "groundwater", "timeseries", "analysis"]
+classifiers = [
+    'Development Status :: 4 - Beta',
+    'Intended Audience :: Science/Research',
+    'Intended Audience :: Other Audience',
+    'License :: OSI Approved :: MIT License',
+    'Programming Language :: Python :: 3 :: Only',
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3.9',
+    'Programming Language :: Python :: 3.10',
+    'Topic :: Scientific/Engineering :: Hydrology',
+]
+
+[project.urls]
+homepage = "https://pastas.dev"
+repository = "https://github.com/pastas/pastas"
+documentation = "https://pastas.readthedocs.io/en/latest/"
+
+[project.optional-dependencies]
+ci = [
+    "pastas[full]",
+    "jupyter",
+    "pytest",
+    "pytest-cov",
+    "codacy-coverage",
+    "corner",
+    "emcee",
+    "tqdm",
+]
+full = ["numba >= 0.51", "lmfit >= 1.0.0"]
+rtd = [
+    "nbsphinx",
+    "Ipython",
+    "ipykernel",
+    "pydata-sphinx-theme",
+    "sphinx-gallery",
+    "sphinx>=3.1, <6.0",
+    "sphinxcontrib-bibtex",
+    "requests",
+]
+develop = ["pastas[full]", "pastas[ci]", "numba-scipy >= 0.3.1"]
```

Comment:
Shouldn't we do something here with numby-scipy support of scipy<=1.7.3 while scipy is already at v1.10.0?

Reply:
Good point, but I'm guessing numba-scipy will cause scipy to be downgraded in this case?

Numba-scipy is also very specific to modfelling multiple wells far so maybe we should just remove mention of it altogether, or move it back into its own little category, like you had before.

Reply:
I'll move it back into its own special group.